pkg/controller/runGC_test.go

```diff
@@ -0,0 +1,197 @@
+package controller
```

Comment:
It might be nice to have a test case for the timestamp ordering: are we properly deleting the oldest X image builds?